bin/openmetadata-server-start.sh

```diff
@@ -111,7 +111,7 @@ fi
 
 #Application classname
 APP_CLASS="org.openmetadata.service.OpenMetadataApplication"
-
+echo $OPENMETADATA_OPTS
```

Comment:
Is this echo intended here? If yes, do we want to add any additional information to this echo line here ?